lib/kubeclient/common.rb

```diff
@@ -204,6 +204,16 @@ def retrieve_all_entities(entity_types)
         end
       end
 
+      def proxy_url(kind, name, port, namespace = '')
+        ns_prefix = build_namespace_prefix(namespace)
+        # TODO: Change this once services supports the new scheme
+        if kind.to_s == 'pods'
```

Comment:
This should probably be: `kind.to_s.pluralize` it can be done in another patch though.